jruby-gradle-base-plugin/src/integTest/groovy/com/github/jrubygradle/JRubyExecIntegrationSpec.groovy

```diff
@@ -105,6 +105,55 @@ class JRubyExecIntegrationSpec extends Specification {
         outputBuffer =~ /Not valid/
     }
 
+    // TODO a second rubygems-servlets test runs into java.lang.OutOfMemoryError: PermGen space 
```

Comment:
Should this be uncommented or just deleted? Doesn't make sense to merge commented out tests IMHO